File: Lenguajes Orientados a Objetos/Actividad 9/Java/src/Empleado.java
public class Empleado {

    String toString(int n)
    {
        return Integer.toString(n);
    }
    // Propiedades, setters y getters
    private String nombre;
    public String getNombre()
    {
        return nombre;
    }
    public void getNombre(String newname)
    {
        this.nombre = newname;
    }

    private int anioContratacion;
    public int getAnioContratacion()
    {
        return anioContratacion;
    }
    public void setAnioContratacion(int a)
    {
        this.anioContratacion = a;
    }

    private double salario;
    public double getSalario()
    {
        return salario;
    }
    public void setSalario(double s)
    {
        this.salario = s;
    }


    // Constructores
    public Empleado()
    {
        this.nombre = "NULL";
        this.anioContratacion = 0;
        this.salario = 0.0;

    }

    Empleado(String n, int anio, double s)
    {
        this.nombre = n;
        this.anioContratacion = anio;
        this.salario = s;

    }

    // Métodos
    public void Imprimir()
    {
        System.out.println("Nombre: " + this.nombre);
        System.out.println("Año de contratación: " + this.anioContratacion);
        System.out.println("Salario: " + this.salario);
    }

    public double CalcularSalario()
    {
        double salario_mensual = this.salario*12;

        System.out.println("Teniendo en cuenta que el salario introducido es mensual, su salario anualmente es de $" + salario_mensual);
        return salario_mensual;
    }

}

File: Lenguajes Orientados a Objetos/Actividad 9/python/empleado.py

class Empleado():
    def __init__(self, nombre="NULL", anioContratacion=0, salario=0.0):
        self.__nombre = nombre
        self.__anioContratacion = anioContratacion
        self.__salario = salario

    @property
    def nombre(self):
        return self.__nombre
    
    @nombre.setter
    def nombre(self, value):
        self.__nombre = value


    @property
    def anioContratacion(self):
        return self.__anioContratacion
    
    @anioContratacion.setter
    def anioContratacion(self, value):
        self.__anioContratacion = value


    @property
    def salario(self):
        return self.__salario
    
    @salario.setter
    def nombre(self, value):
        self.__salario = value


    def Imprimir(self):
        print(f"El nombre del empleado es: {self.__nombre}\nSe le contrato en el anio {self.__anioContratacion}\nSu salario mensual es de {self.__salario}")
        
    def calcularSalario(self):
        print(f"Dado que el salario mensual es de: {self.__salario}, su salario anual seria de {self.__salario*12}")
        return self.__salario*12